test/e2e/network/service_cidrs.go

```diff
@@ -36,7 +36,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = common.SIGDescribe(feature.ServiceCIDRs, func() {
+var _ = common.SIGDescribe(framework.WithFeatureGate(features.MultiCIDRServiceAllocator), func() {
```

Comment:
During the transition, we should tag with:
```suggestion
var _ = common.SIGDescribe(feature.ServiceCIDRs, framework.WithFeatureGate(features.MultiCIDRServiceAllocator), func() {
```

Reply:
done